tools/build.py

```diff
@@ -204,6 +204,7 @@ def path_from_env_var(env_var: str):
         "--build_apple_framework", action="store_true", help="Build a macOS/iOS framework for ONNX Runtime Extensions."
     )
     parser.add_argument("--ios", action="store_true", help="build for iOS")
+    parser.add_argument("--macos", choices=["MacOSX", "Catalyst"])
```

Comment:
nit: is the casing inconsistent? the `--apple-sysroot` values are all lowercase so wondering if these could/should be the same.

Please add `help=...`.

Reply:
Added a`help=...` message.

yeah the `sysroot` are all in lowercase, however here I was just trying to keep it consistent with ORT options.

https://github.com/microsoft/onnxruntime/blob/e02aef1dedc15e9eed9410b54c5024832e9f3164/tools/ci_build/build.py#L404-L408